docs/python/environments.md

```diff
@@ -230,6 +230,12 @@ By default, the debugger will use the Python interpreter you have selected with
 
 For more details on debug configuration, see [Debugging configurations](/docs/python/debugging.md).
 
+### Limited support for Python 2.7
+
+The Python extension no longer offers IntelliSense support for Python 2.7 with [Jedi](https://pypi.org/project/jedi/), as its latest version no longer supports Python 2.7. When using Python 2.7 with the Python extension you can customize the [language server setting](/docs/python/settings-reference#_intellisense-engine-settings) to either turn off auto-completions or select Pylance as your language server, as it may provide a good experience if the code is compatible enough with Python 3.
```

Comment:
```suggestion
The Python extension no longer offers IntelliSense support for Python 2.7 with [Jedi](https://pypi.org/project/jedi/) as it only supports Python 3 at this point. When using Python 2.7 with the Python extension you can customize the [language server setting](/docs/python/settings-reference#_intellisense-engine-settings) to either turn off auto-completions or select Pylance as your language server, as it may provide a good experience if the code is compatible enough with Python 3.
```